BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -78,22 +77,8 @@ class StaticTableViewController: UITableViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        if intrinsicHeightCells {
-            tableView.estimatedRowHeight = 44.0
-        }
```

Comment:
if it doesn’t have estimatedRowHeight set it actually causing infinite loop from numberOfRowInSection which will call heightForRowAt again.
How about we set estimatedRowHeight when user set * intrinsicHeightCells* or *filledUsingLastCell*? changed in latest commit.